app/controllers/admin/users_controller.rb

```diff
@@ -2,7 +2,21 @@ class Admin::UsersController < Admin::BaseController
   inherit_resources
 
   def index
-    @users = params[:show_deleted] ? User.deleted : User.undeleted
+    sort = if params["sort"]
+             case params["sort"]
+             when "name" then "first_name, last_name"
+             when "email" then "email"
+             when "last_activity" then "last_activity"
+             when "name_reverse" then "first_name DESC, last_name DESC"
+             when "email_reverse" then "email DESC"
+             when "last_activity_reverse" then "last_activity DESC"
+             end
+           else
+             "first_name, last_name"
+           end
+
+    @users = params[:show_deleted] ? User.deleted.order(sort) : User.undeleted.order(sort)
```

Comment:
please share the `order(sort)` like:
```ruby
@users = params[:show_deleted] ? User.deleted : User.undeleted
@users = @users.order(sort)
```

or merge it into line 20

Reply:
Done!